kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -208,6 +222,68 @@ apt_get_install()
     wait_for_apt_locks
 }
 
+generate_api_model()
+{
+    touch $DEFINITION_TEMPLATE_NAME
+    cat > $DEFINITION_TEMPLATE_NAME <<EOL
+    {
+    "apiVersion": "vlabs",
+    "location": "",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "",
+            "kubernetesConfig": {
+                "kubernetesImageBase": "",
+                "useInstanceMetadata": false,
+                "networkPlugin": "kubenet",
+                "containerRuntime": "docker",
+                "cloudProviderRateLimitBucket": 0,
+                "cloudProviderRateLimitBucketWrite": 0,
+                "addons": [
+                    {
+                        "name": "tiller",
+                        "enabled": false
+                    }
+                ]
+            }
+        },
+        "customCloudProfile": {
+            "portalURL": ""
+        },
+        "masterProfile": {
+            "dnsPrefix": "",
+            "distro": "ubuntu",
```

Comment:
I would not set any property we later set (ex: distro). I have the impression It may help us to fail fast and catch errors